procurement_auto_create_group/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2017 Eficent Business and IT Consulting Services, S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Procurement Auto Create Group",
+    "version": "12.0.1.0.0",
+    "development_status": "Production/Stable",
+    "license": "AGPL-3",
+    "summary": "Allows to configure the system to propose automatically new "
+               "procurement groups in procurement orders.",
```

Comment:
You need to change this as procurements does not exist anymore

Reply:
Suggestion?